core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserStoreManager.java

```diff
@@ -381,7 +381,9 @@ protected abstract void doUpdateRoleListOfUser(String userName, String[] deleted
                 }
             }
         }
-        log.debug("Retrieving internal roles for user name :  " + userName + " and search filter " + filter);
+        if (log.isDebugEnabled()) {
+            log.debug("Retrieving internal roles for user name :  " + userName + " and search filter " + filter);
```

Comment:
log.debug("Retrieving internal roles for user name :  " + userName + ", and search filter :" + filter);

NOTE the delimiters